commands/sandbox/create.ts

```diff
@@ -65,17 +98,16 @@ exports.handler = async options => {
   let typePrompt;
   let namePrompt;
 
-  if ((type && !SANDBOX_TYPE_MAP[type.toLowerCase()]) || !type) {
+  if ((type && !(type.toLowerCase() in SANDBOX_TYPE_MAP)) || !type) {
     if (!force) {
       typePrompt = await sandboxTypePrompt();
     } else {
       logger.error(i18n(`${i18nKey}.failure.optionMissing.type`));
       process.exit(EXIT_CODES.ERROR);
     }
   }
-  const sandboxType = type
-    ? SANDBOX_TYPE_MAP[type.toLowerCase()]
-    : typePrompt.type;
+
+  const sandboxType = type ? SANDBOX_TYPE_MAP[type] : typePrompt!.type;
```

Comment:
Noticed that you removed the `toLowerCase()` here but not above - I think this could cause a bug if type existed but wasn't in `SANDBOX_TYPE_MAP`